submissions/vovan-zt/app_friends/script.js

```diff
@@ -0,0 +1,185 @@
+﻿const url = 'https://randomuser.me/api/?results=40'
+
+const myForm = document.querySelector('.form'),
+    content = document.querySelector('.content'),
+    resetBtn = document.querySelector('.filter__reset-button'),
+    hamburgerBtn = document.querySelector('.hamburger'),
+    filter = document.querySelector('.filter'),
+    closeBtn = document.querySelector('.filter__close'),
+    container = document.querySelector('.container')
+
+let dataBase = []
+let sortedDataBase = []
+
+async function getResponse() {
+    try {
+        const response = await fetch(url)
+        const data = await response.json()
+        const { results } = await data
+        dataBase = [...results]
+        sortedDataBase = [...dataBase]
+        showCards(dataBase)
+    } catch (e) {
+        console.error(e)
```

Comment:
Try to add a message for your users if loading failed. Not all users check the browser console if something went wrong)